packages/terra-framework-docs/src/terra-dev-site/doc/table/Examples/AutoLayoutTable.jsx

```diff
@@ -1,6 +1,32 @@
+/* eslint-disable react/forbid-dom-props */
 import React from 'react';
 import Table from 'terra-table';
 
+const longText = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.';
+
+const TruncatedText = () => (
+  <div style={{
+    position: 'absolute',
+    left: 0,
+    right: 0,
+    top: 0,
+    bottom: 0,
+    display: 'flex',
+    alignItems: 'center',
+    padding: '0 7px',
```

Comment:
We don't want consumers manipulating the padding, right?  This value is not consistent across all themes.

Reply:
I was concerned about the padding too, but this is an absolutely positioned div inside the cell div, so there will be no padding provided by table and without any padding the text will be aligned to the left (see attachment). 
<img width="214" alt="Screenshot 2024-04-10 at 1 17 40 PM" src="https://github.com/cerner/terra-framework/assets/119358186/e4b2c2fd-48ea-47bc-b9c4-95181101d42b">

Reply:
I don't think we should give consumers this approach as they would have to properly account for themes as well.

Reply:
Changed in this commit: https://github.com/cerner/terra-framework/pull/2131/commits/78c887c9fb73d23f947dea7abe14e3549fc86537